proposals/deliverable-proposals/binding-templates.md

```diff
@@ -0,0 +1,28 @@
+## Bindings Templates
+
+In the next iteration of the binding templates, the group is planning the following work:
+
+1. Evaluation of going for the recommendation track or registry track and adopting one. The choice should allow:
+  1. Versioning of the documents in a stable fashion that allow them to be used in implementations.
+  2. Documentation of the implementations of the individual binding templates.
+  3. Encourage new binding templates to be written by different (working group) members.
+  4. Publication of ontologies of protocols
+2. Publication of the currently existing binding templates (as registry or recommendation depending on the point above):
+  1. HTTP 
+  2. CoAP
+  3. MQTT
+  4. Modbus
+3. Starting work on the new binding template documents and publishing the first working draft of each
+  1. Protocols
+    1. BACnet
+    2. gRPC
+    3. OPC-UA (also see https://github.com/w3c/wot/blob/main/liaisons/opcf/tech_reqs.md)
+  2. Payload Formats
+    1. JSON
```

Comment:
CBOR?

Reply:
Done

Reply:
I am not opposed to see "CBOR" on the list... but in my understanding JSON, XML are formats and there are default *textual* serializations of it. On the the other hand there a more efficient serializations like CBOR for the JSON format...

Reply:
I also think that CBOR can be "simply used". This binding should simply show its usage via examples. I would not expect that we define an ontology for it. @ektrah any opinions?

Reply:
I agree that CBOR is useful to have. I'm not sure if I understand the rest of the question... Let's say there is a Thing that exposes a Property value in [application/senml+cbor](https://www.rfc-editor.org/rfc/rfc8428) format over CoAP. What's needed (in terms of payload bindings) to make that work?

Reply:
If you check out [section 6](https://www.rfc-editor.org/rfc/rfc8428#section-6) (and in particular table 4) of RFC 8428, you'll see that `application/senml+cbor` is not just `application/senml+json` encoded into CBOR. Would that table need to go into the Thing Description? Or would we need a separate SenML payload binding?

Reply:
IMHO, if the goal is to convert a JSON payload (as described by the data schema in JSON Schema terms) into some random CBOR structure, then, yes, you can simply encode it into CBOR. However, if the goal is to convert it into the CBOR structure that people actually use, I think a different approach is needed: The payload binding would need to specify (1.) how to express arbitrary CBOR/XML/... payloads in JSON and (2.) how to convert the CDDL/XML Schema/... that describes the CBOR/XML/... payload to JSON Schema.